### PR TITLE
allow score and stderr to be zero (vs. null)

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/model/Measures.java [hunk 1]
```diff
@@ -9,22 +9,22 @@
...
+    private Integer studentCount;
 
     public Integer getAvgScaleScore() {
-        return (avgScaleScore != null && avgScaleScore == 0) ? null : avgScaleScore;
+        return avgScaleScore;
     }
 
     public Integer getAvgStdErr() {
-        return (avgStdErr != null && avgStdErr == 0) ? null : avgStdErr;
+        return avgStdErr;
```
Comment (malaffoon) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/model/Measures.java]: Don't conflate null and 0. But now we have to make sure any place we set it we know whether it should be null or zero. There are only three places that seem to set the values ...

File: aggregate-service/src/main/java/org/opentestsystem/rdw/olap/repository/impl/AbstractReportRepository.java [hunk 2]
```diff
@@ -183,8 +183,8 @@ protected void validate(final Q query) {
 
     protected Measures measures(final ResultSet row, final String asmtTypeCode, final String prefix) throws SQLException {
         final Measures.Builder builder = Measures.builder()
-                .avgScaleScore(row.getInt(prefix + "score"))
-                .avgStdErr(row.getInt(prefix + "std_err"))
+                .avgScaleScore(getNullable(row, row.getInt(prefix + "score")))
+                .avgStdErr(getNullable(row, row.getInt(prefix + "std_err")))
```
Comment (malaffoon) [inline aggregate-service/src/main/java/org/opentestsystem/rdw/olap/repository/impl/AbstractReportRepository.java]: recall that `row.getInt` returns a 0 if the value is null ... we can't have that

Comment (esotrope) [reply aggregate-service/src/main/java/org/opentestsystem/rdw/olap/repository/impl/AbstractReportRepository.java]: nice find!

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/model/Measures.java (hunk 1 above)
Comment (esotrope) [reply common/src/main/java/org/opentestsystem/rdw/reporting/common/model/Measures.java]: the UI the only thing that consumes these values? or does the backend process them more anywhere?

Comment (malaffoon) [reply common/src/main/java/org/opentestsystem/rdw/reporting/common/model/Measures.java]: I don't see it being processed anywhere else ...

**Pull request merged**